app/microservice/register.json

```diff
@@ -56,6 +56,15 @@
             "baseUrl": "#(service.uri)",
             "path": "/api/v1/subscriptions/find-by-ids"
         }]
+    }, {
+        "url": "/v1/subscriptions/admin/find-all",
```

Comment:
Not sure I like the `admin` prefix. For example, `/find-by-idy` is virtually equivalent to this, and does not have a special prefix

Reply:
Initially I had some issues with coliding route names, but now updated the route name to just `/find-all`